src/server/scripts/ExilesReach/zone_exiles_reach.cpp

```diff
@@ -5330,6 +5330,562 @@ class quest_choppy_booster_mk5 : public quest_scout_chopper
+        if (!player)
+            return;
+
+        Conversation* conversation = Conversation::CreateConversation(12086, player, *player, player->GetGUID(), nullptr, false);
+        if (!conversation)
+            return;
+
+        conversation->AddActor(0, 0, player->GetGUID());
+        conversation->AddActor(ACTOR_LINDIE_RESIZING_QUEST, 1, me->GetGUID());
+        conversation->AddActor(ACTOR_CORK_RESIZING_QUEST, 2, ObjectGuid::Empty);
+        conversation->Start();
+
+        _events.ScheduleEvent(EVENT_RESIZING_FOLLOW_PLAYER, 2s);
+    }
+
+    void MovementInform(uint32 uiType, uint32 uiId) override
+    {
+        if (uiType != POINT_MOTION_TYPE)
+            return;
+
+        if (uiId != 0)
+            return;
+
+        if (Player* player = me->GetOwner()->ToPlayer())
+            player->CastSpell(player, SPELL_LINDIE_DESUMMON_Q56034);
+    }
+
+    void SpellHit(WorldObject* /*caster*/, SpellInfo const* spellInfo) override
+    {
+        if (spellInfo->Id != SPELL_PING_LINDIE_Q56034_Q59941)
+            return;
+
+        _events.ScheduleEvent(EVENT_RESIZING_RUN_HOME, 2s);
+    }
+
+    void UpdateAI(uint32 diff) override
+    {
+        _events.Update(diff);
+
+        while (uint32 eventId = _events.ExecuteEvent())
+        {
+            switch (eventId)
+            {
+                case EVENT_RESIZING_FOLLOW_PLAYER:
+                {
+                    if (Unit* owner = me->GetOwner())
+                        me->GetMotionMaster()->MoveFollow(owner, 0.0f, float(M_PI / 4.0f));
+                    break;
+                }
+                case EVENT_RESIZING_RUN_HOME:
+                {
+                    me->GetMotionMaster()->MovePoint(POINT_HOME_POSITION, ResizingGuardianPosition);
+                    break;
+                }
+                default:
+                    break;
+            }
+        }
+    }
+
+private:
+    EventMap _events;
+};
+
+// Entry 167915 - Cork Fizzlepop
+struct npc_cork_fizzlepop_q59941 : public ScriptedAI
+{
+    npc_cork_fizzlepop_q59941(Creature* creature) : ScriptedAI(creature) { }
+
+    void JustAppeared() override
+    {
+        Player* player = me->GetOwner()->ToPlayer();
+        if (!player)
+            return;
+
+        Conversation* conversation = Conversation::CreateConversation(12086, player, *player, player->GetGUID(), nullptr, false);
+
+        if (!conversation)
+            return;
+
+        conversation->AddActor(0, 0, player->GetGUID());
+        conversation->AddActor(ACTOR_LINDIE_RESIZING_QUEST, 1, ObjectGuid::Empty);
+        conversation->AddActor(ACTOR_CORK_RESIZING_QUEST, 2, me->GetGUID());
+        conversation->Start();
+
+        _events.ScheduleEvent(EVENT_RESIZING_FOLLOW_PLAYER, 2s);
+    }
+
+    void MovementInform(uint32 uiType, uint32 uiId) override
+    {
+        if (uiType != POINT_MOTION_TYPE)
+            return;
+
+        if (uiId != 0)
+            return;
+
+        if (Player* player = me->GetOwner()->ToPlayer())
```

Comment:
crash if owner is null (e.g. via `.npc add temp`), also casting to player is not required here